content/admin/sys_settings/SysSettingsGlance.md

```diff
@@ -31,7 +31,7 @@ Get a glimpse into license allocation, see if any licenses are nearing expiratio
 Manage the library compliance policy for your organization(s). Setup restrictions or version requirements for library usage which will flag violations that break compliance settings.
 
 ## Security
-Manage [password policies](admin-systemsettings.html#pwd), [two-step verification](admin-orgsecurity.html#security-tsv), and [API and application key management](admin-orgsecurity.html#key-manage). Any change to this setting will require a system restart. 
+Manage [password policies](admin-systemsecurity.html#pwd), [two-step verification](admin-orgsecurity.html#security-tsv), and [API and application key management](admin-systemsecurity.html#manage-api). Any change to this setting will require a system restart. 
```

Comment:
IMO system restart sounds like computer restart/reboot

Reply:
@cgtwin As opposed to the agents being reconfigured?